apps/examples/testcase/ta_tc/security/utc/utc_crypto.c

```diff
@@ -59,28 +59,64 @@ static security_handle g_hnd = NULL;
  * @brief            encrypt AES
  * @scenario         encrypt AES
  * @apicovered       crypto_aes_encryption
- * @precondition     none
+ * @precondition     AES key should be set in Factory 
  * @postcondition    none
  */
 static void utc_crypto_aes_encryption_p(void)
 {
-	int i = 0;
-	unsigned char iv[] = "tamp_iv_value";
-	unsigned int iv_len = strlen((const char*)iv) + 1;
+	unsigned char iv[] = { 0x1a, 0x2b, 0x3c, 0x4d, 0x5e, 0x6f, 0x70, 0x81, 0x92, 0xa3, 0xb4, 0xc5, 0xd6, 0xe7, 0xf8, 0x00 };
+	unsigned int iv_len = 16;
 
-	unsigned char plain_text[] = "plain text";
-	unsigned int plain_text_len = strlen((const char*)plain_text) + 1;
+	unsigned char plain_text[] = { 0x4d, 0x79, 0x20, 0x42, 0x79, 0x74, 0x65, 0x20, 0x50, 0x72, 0x69, 0x6e, 0x74, 0x00, 0x00, 0x00 };
+	unsigned int plain_text_len = 16;
+
+	security_data plain = {plain_text, plain_text_len};
+	security_data enc = {NULL, 0};
+
+	for (int i = 0; i < sizeof(g_aes_mode_table)/sizeof(security_aes_mode); i++) {
+		/* Check whether hang occurs during repeated encrypt function */
+		for (int j = 0; j < 10; j++) {
+			security_aes_mode mode = g_aes_mode_table[i];
+			security_aes_param param = {mode, iv, iv_len};
+
+			security_error res = crypto_aes_encryption(g_hnd, &param, UTC_CRYPTO_KEY_NAME,
+													&plain, &enc);
```

Comment:
When do you change a line? You don't do in line 70.

Reply:
The existing alignment was like this, there was no intent. If I need to align, I'll change it.

Reply:
Anything is ok. But let's use the same way in the same file at least.